python-threatexchange/threatexchange/cli/fetch.py

```diff
@@ -74,7 +74,7 @@ def __init__(
         # Print first update after 5 seconds
         self.last_update_printed = time.time() - self.PROGRESS_PRINT_INTERVAL_SEC + 5
         self.processed = 0
-        self.counts = collections.Counter()
+        self.counts: t.Dict[str, int] = collections.Counter()
```

Comment:
Always thought inferred typing for stdlib would take care of this.

Reply:
me too but mypy didn't agree